libs/clients/passports/src/lib/passportsApi.service.ts

```diff
@@ -220,9 +224,12 @@ export class PassportsService {
     }
   }
 
-  async getCurrentPassport(user: User): Promise<Passport> {
-    const userPassports = await this.getIdentityDocument(user)
-    const childPassports = await this.getIdentityDocumentChildren(user)
+  async getCurrentPassport(
+    user: User,
+    type?: string | null,
```

Comment:
Same as above

Reply:
done with all of those, only one available parameter at the moment, but I put it in a type